server/matchmaker/matchmaker_queue.py

```diff
@@ -152,14 +154,27 @@ async def find_matches(self) -> None:
         if self.num_players < 2 * self.team_size:
             return
```

Comment:
Note to self, @BlackYps pointed out that before returning early here we should be calling `register_failed_matching_attempt`

Reply:
Or I guess use your function `_register_unmatched_searches`